server/test/com/cloud/vpc/MockConfigurationManagerImpl.java

```diff
@@ -456,6 +456,15 @@ public void createDefaultSystemNetworks(long zoneId) throws ConcurrentOperationE
     }
 
     /* (non-Javadoc)
+     * @see com.cloud.configuration.ConfigurationManager#deleteDomainSpecificVirtualRanges(long)
+     */
+    @Override
```

Comment:
@ustcweizhou The method "com.cloud.configuration.ConfigurationManager.deleteDomainSpecificVirtualRanges(long)" does not exist. Could you please alter this?

I am wondering, why this is a "non-Javadoc"? In my point of view, it could be a Javadoc block.

There is another point citing an inexistent method (at "releaseAccountSpecificVirtualRanges" method which is referring to "com.cloud.configuration.ConfigurationManager.deleteAccountSpecificVirtualRanges(long)"). Although this comment is out of this PR scope, I think that it would be nice to take the opportunity.